iot-hub/Samples/service/ImportExportDevicesSample/Parameters.cs

```diff
@@ -6,51 +6,59 @@ namespace Microsoft.Azure.Devices.Samples
     /// <summary>
     /// Configurable parameters for the sample.
     /// </summary>
+    /// <remarks>
+    /// To get these connection strings, log into https://azure.portal.com, go to Resources, open the IoT hub, open Shared Access Policies, open iothubowner, and copy a connection string.
```

Comment:
Reducing how much is printed in help, especially stuff that is duplicated and that the user likely already knows how to do.